quick-protobuf/examples/pb_rs/data_types.rs

```diff
@@ -112,26 +121,62 @@ pub struct FooMessage<'a> {
     pub test_oneof: mod_FooMessage::OneOftest_oneof<'a>,
 }
 
+impl<'a> FooMessage<'a> {
+    pub const DEFAULT_F_SINT64: i64 = 4i64;
+    pub const DEFAULT_F_BOOL: bool = true;
+    pub const DEFAULT_F_FIXED32: u32 = 0u32;
```

Comment:
As I mentioned in the issue, I think it is worth generating a getter for all fields that have a default value in the protobuf file.

If there are concerns about the generated code size, we can hide it behind a config flag so it can be turned off/on although I am not sure it is worth the complexity. If you have a proto file with explicit defaults, there is a good chance you will have to type the code to access it so we might as well generate it for the user.

Reply:
Yep, working on this currently -- this is the main holdup as the codegen is getting a bit fiddly